test/files/sample.go

```diff
@@ -4,56 +4,61 @@
 package sample1
 
 import (
```

Comment:
I think `gofmt` took a hold of this file, which is why the diff is so big. Only addition should be the `Woop` field on `R`.